flink/v1.16/flink/src/test/java/org/apache/iceberg/flink/sink/TestTaskWriters.java

```diff
@@ -53,16 +53,17 @@ public class TestTaskWriters {
   private static final long TARGET_FILE_SIZE = 128 * 1024 * 1024;
 
   @Rule public final TemporaryFolder tempFolder = new TemporaryFolder();
+  private final String branch;
 
   @Parameterized.Parameters(name = "format = {0}, partitioned = {1}")
   public static Object[][] parameters() {
     return new Object[][] {
-      {"avro", true},
-      {"avro", false},
-      {"orc", true},
-      {"orc", false},
-      {"parquet", true},
-      {"parquet", false}
+      {"avro", true, "main"},
```

Comment:
we probably shouldn't need to add the branch parameter here. this class is mainly for testing Flink TaskWriter. branching or not is not really important.